src/main/java/pl/wedding/management/service/reservation/domain/converter/ReservationRowFactory.java

```diff
@@ -0,0 +1,32 @@
+package pl.wedding.management.service.reservation.domain.converter;
+
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import pl.wedding.management.service.converter.RowFactory;
+import pl.wedding.management.service.reservation.domain.entity.ReservationEntity;
+import pl.wedding.management.service.reservation.domain.entity.ReservationDto;
+
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Optional;
+
+@NoArgsConstructor
+public class ReservationRowFactory implements RowFactory<ReservationDto, ReservationEntity> {
+
+    @Override
+    public ReservationEntity convertFromDto(@NonNull ReservationDto dto) {
+        return new ReservationEntity(dto.getId(),
+                getDateTime(dto.getReservationStart()),
+                getDateTime(dto.getReservationEnd()),
+                dto.getWeddingHall(),
+                dto.getTenant(),
+                dto.getNumberOfGuests(),
+                dto.getOccasion());
+    }
+
+    private LocalDateTime getDateTime(LocalDateTime dateTime) {
+        return Optional.ofNullable(dateTime)
+                .map(input -> input.truncatedTo(ChronoUnit.MINUTES))
```

Comment:
Nie powinniśmy modyfikować w żaden sposób danych wczytywanych z bazy danych. Z założenia powinno je się obsługiwać w taki sposób, w jaki są zapisane. Jeśli będą złe - trudno. To znaczy, że coś poszło nie tak na wcześniejszym etapie.